growth/too/models.py

```diff
@@ -435,8 +437,11 @@ class Field(db.Model):
 
     field_id = db.Column(
         db.Integer,
+        Sequence('field_id_seq', start=10000001, increment=1),
```

Comment:
Is it possible to count down from zero to avoid any possible collision with pre-existing fields?